x-pack/legacy/plugins/monitoring/server/telemetry/get_stats_with_monitoring.ts

```diff
@@ -16,7 +17,7 @@ import { getAllStats, getLocalStats } from './';
  * @param {Boolean} unencrypted Is the request payload going to be unencrypted.
  * @return {Promise} An array of telemetry objects.
  */
-export async function getStats(
+export async function getStatsWithMonitoring(
```

Comment:
Since the monitoring plugin is responsible for registering this collector, it shouldn't be necessary to check for `config.get('xpack.monitoring.enabled')` on line 32. That way monitoring also doesn't have to import OSS `_getLocalStats`.